### ISSUE TITLE
[dataclasses] I3Time printing bug (Trac #1551)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1543]
First reported by jbellinger.  Here's an example:

```text
In [1]: print I3EventHeader.start_time
2011-05-16 00:00:00.011,197,723,5 UTC

In [2]: I3EventHeader.start_time
Out[2]: I3Time(2011,116640000117197235L)
```

It's dividing by an extra factor of 10 here: http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/projects/dataclasses/trunk/private/dataclasses/I3Time.cxx#L451

Migrated from https://code.icecube.wisc.edu/ticket/1551
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:26", 
    "description": "First reported by jbellinger.  Here's an example:\n\n{{{\nIn [1]: print I3EventHeader.start_time\n2011-05-16 00:00:00.011,197,723,5 UTC\n\nIn [2]: I3EventHeader.start_time\nOut[2]: I3Time(2011,116640000117197235L)\n}}}\n\nIt's dividing by an extra factor of 10 here: http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/projects/dataclasses/trunk/private/dataclasses/I3Time.cxx#L451\n", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067086520250", 
    "component": "combo core", 
    "summary": "[dataclasses] I3Time printing bug", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2016-02-16T18:11:51", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160216T20:38:35: 
* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "new" to "closed"
* david.schultz commented:

> In changeset 141966:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="141966"
> fix I3Time printing bug. closes #1543
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:26: 
* olivas commented:

> Milestone offline V16-03-00 deleted


* olivas changed milestone from "offline V16-03-00" to ""
